aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -122,6 +126,38 @@ def save(self, names=()) -> None:
         self.names = self.omx_file.list_matrices()
         self.computational_view(names)
 
+    def __save_as(self, file_name: str, cores: List[str]):
+
+        if ".aem" in file_name:
+            mat = AequilibraeMatrix()
+            args = {
+                "zones": self.zones,
+                "matrix_names": cores,
+                "index_names": self.index_names,
+                "memory_only": False,
+                "file_name": file_name,
+            }
+            mat.create_empty(**args)
+            mat.indices[:, :] = self.indices[:, :]
+            for core in cores:
+                mat.matrix[core][:, :] = self.matrix[core][:, :]
+            mat.name = self.name
+            mat.description = self.description
+            mat.close()
+            del mat
+
+        elif ".omx" in file_name:
```

Comment:
Ideally we'd be using pathlib's `suffix` to handle this. A path can contain `.omx` or `.aem` where that's not the extension. One day we should move over to just using pathlib everywhere to avoid messy strings 
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix

Reply:
Agreed and fixed